.chloggen/remove-deprecated-cpu-metric.yaml

```
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

We already deprecated this metric, this is a breaking change:

```suggestion
change_type: breaking
```